js/index.js

```diff
@@ -52,17 +54,17 @@ leave_message.addEventListener('submit',function(e){
     let messageSection = document.getElementById('messages');
     let messageList = document.querySelector('ul');
     let newMessage = document.createElement ('li');
-   newMessage.innerHTML=`<a href='mailto:asel.karagazieva@gmail.com'>${Name}</a>
+   newMessage.innerHTML =`<a href='mailto:asel.karagazieva@gmail.com'>${Name}</a>
    wrote <span>${Message}</span>`
    messageSection.appendChild(newMessage);
 
-    let removeButton=document.createElement('button')
-    removeButton.id="buttonremove";
+    let removeButton = document.createElement('button')
+    removeButton.id = "buttonremove";
     removeButton.innerHTML = 'remove';
     removeButton.type = 'button';
     messageSection.appendChild(removeButton)
-    removeButton.addEventListener('click', (e) =>{
-    let entry= document.getElementById('messages').parentNode
+    removeButton.addEventListener('click', (e) => {
+    let entry = document.getElementById('messages').parentNode
```

Comment:
Yeah the extra little work you do here for readability is 👌 . In the future you'll probably be introduced to linters that do this for you automatically, but appreciate the effort here.